### PR TITLE
Document WebRTC streamer configuration

### PR DESCRIPTION
**Pull request by morphis**

### CODE REVIEW

File: ref/webrtc-streamer.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+| `max_kbps` | integer | 0 | Maximum bitrate in kb/s |
+
+## Example configuration
+
+The following shows an example configuration for the WebRTC streamer:
+
+    {
+        "video": {
+            "bitrate_limits": [
+                { "num_pixels": 921600, "min_kbps": 1000, "max_kbps": 4500 },
```
Comment (adglkh) [inline ref/webrtc-streamer.md:40]: One suggestion: can we briefly explain how people can give a decent num_pixels as a baseline, which can be tweaked later? E.g. if the num_pixels < encoder.frame.width * encoder.frame.height, the bitrate limitation will be ignored.

Comment (morphis) [reply ref/webrtc-streamer.md:40]: How about now? (see below)

Comment (adglkh) [reply ref/webrtc-streamer.md:40]: Looks good.
Thank!

**Review by adglkh — APPROVED** [review]

**Review by ru-fu — CHANGES_REQUESTED** [review]
I'll add a commit with some changes.

**Comment by ru-fu** [comment]
@morphis Please check if you're fine with the changes.

**Review by ru-fu — APPROVED** [review]

**Pull request merged**